depends/packages/qt.mk

```diff
@@ -70,19 +71,35 @@ $(package)_config_opts += -system-zlib
 $(package)_config_opts += -static
 $(package)_config_opts += -silent
 $(package)_config_opts += -v
+$(package)_config_opts += -no-feature-bearermanagement
+$(package)_config_opts += -no-feature-colordialog
+$(package)_config_opts += -no-feature-commandlineparser
+$(package)_config_opts += -no-feature-concurrent
```

Comment:
I was scared here for a bit, but turns out `concurrent` is for high-level thread handling, an PI that is not used in our project, and it's not necessary for low-level multithreading that we use.